src/Tiamat_oxDNA.py

```diff
@@ -313,22 +366,24 @@ def print_usage():
         print("\t[-m\--molecule=DNA|RNA]", file=sys.stderr)
         print("\t[-t\--tiamat-version=1|2]", file=sys.stderr)
         print("\t[-t\--default-base=A|C|G|T|R|i (R = random, i = any integer)]", file=sys.stderr)
-        print("\t[-f\--print-force-file]\n", file=sys.stderr)
+        print("\t[-f\--print-force-file]", file=sys.stderr)
+        print("\t[-o\--print-oxview]\n", file=sys.stderr)
         print("\tThe defaults options are --molecule=DNA, --tiamat-version=2, --default-base=R", file=sys.stderr)
         exit(1)
 
 
 def parse_options():
-    shortArgs = 'd:m:t:f'
-    longArgs = ['default-base=', 'molecule=', 'tiamat-version=', 'print-force-file']
+    shortArgs = 'd:m:t:f:o'
+    longArgs = ['default-base=', 'molecule=', 'tiamat-version=', 'print-force-file', 'print-oxview']
     
     # for some reason, files originally made in T1 have a different .json form than T2
     # it would be possible to rewrite all the parameters to fix it, but tossing a factor
     # of 1.2 for DNA and 1.6 for RNA on base_vector seems to be good enough
     opts = {
         "isDNA" : True,
         "tiamat_version_fudge" : 1,
-        "print_force_file" : False
+        "print_force_file" : False,
+        "print-oxview" : False
```

Comment:
I'm being really obnoxious here, but could you use print_oxview to keep it aligned with the other options?

Reply:
Thanks for spotting this, I just named it the same as the argument and missed the convention for the others. Fixed now